test/e2e/serial/tests/workload_placement_tmpol.go

```diff
@@ -1401,14 +1398,14 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 					loggedEvents = true
 				}
 				return false
-			}).WithTimeout(2*time.Minute).WithPolling(10*time.Second).Should(BeTrue(), "pod %s/%s doesn't contains the expected event error", updatedPod.Namespace, updatedPod.Name)
+			}).WithTimeout(2*time.Minute).WithPolling(10*time.Second).Should(BeTrue(), "pod %s/%s doesn't contains the expected event error", pod.Namespace, pod.Name)
 
 			By("deleting the test pod")
-			err = fxt.Client.Delete(context.TODO(), updatedPod)
+			err = fxt.Client.Delete(context.TODO(), pod)
 			Expect(err).ToNot(HaveOccurred())
 
 			By("checking the test pod is removed")
-			err = wait.With(fxt.Client).Timeout(3*time.Minute).ForPodDeleted(context.TODO(), updatedPod.Namespace, updatedPod.Name)
+			err = wait.With(fxt.Client).Timeout(3*time.Minute).ForPodDeleted(context.TODO(), pod.Namespace, pod.Name)
```

Comment:
all these changes seems unnecessary if we retrieve `updatedPod` like I suggested above. If we don't, I'm not sure why this code is more correct than before

Reply:
answered in https://github.com/openshift-kni/numaresources-operator/pull/950#discussion_r1668432441